doc/config.md

```diff
@@ -650,6 +650,8 @@ Simple do-nothing query to check if the server connection is alive.
 
 If an empty string, then sanity checking is disabled.
 
+If `<empty>` then send empty query as sanity check.
+
 Default: `select 1`
```

Comment:
The default is not changed anymore, was that intended? I think it might be fine to change the default to `<empty>`, but we need some test coverage in that case. Because currently we don't seem to have any `server_check` tests.